javaslang/src/test/java/javaslang/collection/LinkedHashMapTest.java

```diff
@@ -69,8 +69,14 @@ protected String className() {
 
     @Test
     public void shouldKeepOrder() {
-        CharSeq actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(CharSeq.empty(), (s, t) -> s.append(t._2));
-        assertThat(actual).isEqualTo(CharSeq.of("abc"));
+        List<Character> actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(List.empty(), (s, t) -> s.append(t._2));
+        Assertions.assertThat(actual).isEqualTo(List.of('a', 'b', 'c'));
```

Comment:
> This is the reason we did not recognize a bug...

No, this bug was not recognized because of there was not test for it :) But we could not notice the bug in `fold()`, if it were